birdhouse/read-configs.include.sh

```diff
@@ -147,7 +149,7 @@ read_components_default_env() {
 process_delayed_eval() {
     ALREADY_EVALED=''
     for i in ${DELAYED_EVAL}; do
-        if echo "ALREADY_EVALED" | grep -qE "^\s*$i\s*$"; then
+        if echo "$ALREADY_EVALED" | grep -qE "^\s*$i\s*$"; then
```

Comment:
LOL, I made this same kind of mistake often too :)

Reply:
I'm a little embarrassed by that one :disappointed: